tests/metrics/test_kbet.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+from tests.common import *
+
+
+def test_kbet(adata_pca):
+    score = scIB.me.kBET(
+        adata_pca,
+        batch_key='batch',
+        label_key='celltype',
+        embed='X_pca'
+    )
+    score = 1 - np.nanmean(score['kBET'])
+    LOGGER.info(f"score: {score}")
+    assert np.isnan(score)
```

Comment:
I think `np.nanmean` ensures this is never nan. Apparently a runtime warning is output if `np.nanmean` gets a vector of all `NaN`s.

Reply:
For small cell types, where kBET might not be applicable, a `NaN` value is returned. `np.mean` fails when it encounters `NaN`, `np.nanmean` ignores them.

Reply:
This test prob also needs to work when kBET can't be run on any celltype... This might be the case, no?

Reply:
Indeed. In the case of all `NaN`, `np.nanmean` returns only `nan`.

Reply:
Ah... wait... a RunTimeWarning doesn't mean an error. All good here.